templates/header/topbar.html

```diff
@@ -25,26 +25,73 @@
 
                 <ul class="pure-menu-list">
                     <li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover">
-                        <a href="/about" class="pure-menu-link disable-menu-arrow">
+                        <a href="/about" class="pure-menu-link">
                             About
                         </a>
 
                         <ul class="pure-menu-children">
-                            <a href="/about" class="pure-menu-link">
-                                Documentation about builds, badges and more
-                            </a>
+                            <li class="pure-menu-item">
+                                <a href="/about/badges" class="pure-menu-link">
+                                    Badges
+                                </a>
+                            </li>
```

Comment:
Can you use the `active_link` macro here?
```suggestion
                            {{ macros::active_link(expected="never", href="/about/badges", text="Badges") }}
```
Same for all the following links.

Reply:
I *might* have gone overboard